src/xcode/ENA/ENA/Source/HTTPClient/Resources/AvailableHours/AvailableHoursResource.swift

```diff
@@ -0,0 +1,50 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct AvailableHoursResource: Resource {
+
+	// MARK: - Init
+
+	init(
+		day: String,
+		country: String,
+		trustEvaluation: TrustEvaluating = DefaultTrustEvaluation(
+			publicKeyHash: Environments().currentEnvironment().pinningKeyHashData
+		)
+	) {
+		self.locator = .availableHours(day: day, country: country)
+		self.sendResource = Send()
+		self.receiveResource = Receive()
+		self.trustEvaluation = trustEvaluation
+		self.type = .wifiOnly
+	}
+
+	// MARK: - Protocol Resource
+
+	typealias Send = EmptySendResource
+	typealias Receive = JSONReceiveResource<[Int]>
+	typealias CustomError = Error
+
+	var locator: Locator
+	var type: ServiceType
+	var sendResource: EmptySendResource
+	var receiveResource: JSONReceiveResource<[Int]>
+	var trustEvaluation: TrustEvaluating
+
+	var defaultModelRange: [Int] {
+		[404]
```

Comment:
Do we have a tech spec we can link here to explain why?

Reply:
I copied over the old comment - I'll search for a tech spec